### ISSUE TITLE
Machine Learning Performance Page

### ISSUE THREAD

**Issue by alexrobincrabbe** [alexrobincrabbe/Brain_tumour_diagnostic#14]
As a use I can user I can view the machine learning performance plots so that I can assess the performance of the model

### Acceptance Criteria

- Machine learning performance page is added to the streamlit app
- the loss and accuracy curves are shown

### Tasks

- [x] Create function to generate content for machine learning performance page
- [x] Show loss and accuracy curves for training and validation sets
- [x] Show metrics on test set
- [x] Show precision and recall curves for training and validation set, for each label
